NAMESPACE

```diff
@@ -691,10 +719,12 @@ importFrom(rlang,":=")
 importFrom(rlang,quo)
 importFrom(rlang,sym)
 importFrom(rlang,syms)
+importFrom(shiny,reactivePoll)
```

Comment:
Note previous comment that I think shiny should be a `Suggests` so we shouldn't import this directly (rather do `requireNamespace` and call via `shiny::rectivePoll()`